samples/bot-calling-meeting/csharp/Source/CallingBotSample/Bots/MessageBot.cs

```diff
@@ -256,17 +229,104 @@ private async Task SendResponse(ITurnContext<IMessageActivity> turnContext, stri
             }
         }
 
-        private async Task SendMeetingActionsCard(ITurnContext turnContext, CancellationToken cancellationToken)
+        private async Task<Call> JoinScheduledMeeting(ITurnContext turnContext, CancellationToken cancellationToken)
         {
-            var conversationStateAccessors = conversationState.CreateProperty<MeetingActionDetails>(nameof(MeetingActionDetails));
-            var conversationData = await conversationStateAccessors.GetAsync(turnContext, () => new MeetingActionDetails());
+            var users = await TeamsInfo.GetPagedMembersAsync(turnContext, cancellationToken: cancellationToken);
 
-            var resourceResponse = await turnContext.SendActivityAsync(MessageFactory.Attachment(adaptiveCardFactory.CreateMeetingActionsCard()));
-            conversationData.MeetingActionsCardActivityId = resourceResponse.Id;
-            await conversationState.SaveChangesAsync(turnContext, false, cancellationToken);
+            var organiser = new Identity
+            {
+                // This needs to be the organiser of the meeting, so you can't use the activity invoker
```

Comment:
Is the first user guaranteed to be the organiser?

Reply:
I couldn't find an answer, but found a way to confirm who the organiser was without using RSC.